README.md

```diff
@@ -52,31 +53,19 @@ Make sure, you have at least 1.5 GB of free diskspace.
     ```
 
 
-5.  If you want to save some disk space, you can now clean up the build directory:
+    > The build process takes a while, you can continue with steps 5. to 8. in a
+    > second shell as the build process continues.
```

Comment:
I'd be fine with adding just a remark like this. But I don't like how you are restructuring the entire README here for the purpose of parallel processing. That's just optimizing for the wrong case.